### DISCUSSION TITLE
dpgen simplify parameters

### DISCUSSION THREAD

**Discussion by ghost** [deepmodeling/dpgen#1414] [General]
This is my whole DP-GEN repository.

https://github.com/paseri3739/DP-GEN_Sodium

Tree has directory tree. log is a dpgen simplify log file. simplify.yaml is a simplify parameters file.

In the log, dpgen reports no error but there is no file in ./simplify/ 

I want to simplify iter.*/02.fp/data.000 to ./simplify by both npy and raw, but I have no idea of it. How do I specify the iteration files and simplified file in simplify.yaml ?

**Comment by njzjz** [comment]
It has been supported by #730. You need to update the version.
